osu.Desktop/Program.cs

```diff
@@ -106,5 +132,39 @@ private static bool handleException(Exception arg)
 
             return continueExecution;
         }
+
+        private static object onLegacyIpcMessageReceived(object message)
```

Comment:
Can this not go inside the legacy ipc class?

Reply:
It could. I'll move it there but we'll have to see how this class ends up, because as I said we could e.g. integrate AIMod whilst lazer doesn't have beatmap submission, which may require additional game components.

Reply:
I would not see that happening. I am strongly against anything beyond difficulty calculation.